src/client/dfuse/SConscript

```diff
@@ -173,6 +173,22 @@ def configure_fuse(cenv):
     check.Finish()
 
 
+def find_static_fuse(cenv):
+    """Find and copy static fuse library"""
+
+    fuse_prefix = '/usr'
+    if os.path.exists(cenv.subst('$FUSE_PREFIX')):
+        fuse_prefix = cenv.subst('$FUSE_PREFIX')
+
+    for path in ['lib64', 'lib/x86_64-linux-gnu', 'lib']:
+        fuselib = os.path.join(fuse_prefix, path, "libfuse3.a")
+        if os.path.exists(fuselib):
+            return [File(fuselib)]
+
+    print("Could not find libfuse3.a")
```

Comment:
I think aborting is probably the better option here, it's likely the build would fail immediately afterwards and by aborting then you get to control the error message.  In general if you ask software to do something and it can't then it should not try and resolve the issue by attempting something else.

Reply:
prefer to handle in later patch.  Ultimately, it will fail to build